packages/xo-server/src/xapi/index.js

```diff
@@ -474,6 +474,16 @@ export default class Xapi extends XapiBase {
     await this.call('host.restart_agent', this.getObject(hostId).$ref)
   }
 
+  async setHostIscsi(hostId, iscsi) {
```

Comment:
You can set it in `setHostProperties`.

Reply:
[`setHostProperties`  call `set`'s of `xapi`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/xapi/index.js#L231toL255).

for us we get `iscsi_iqn` from `other_config` we don't use `iscsi_iqn` props.
What do you think we expose `iscsi_iqn` from props and from `other_config` on [xapi-object-to-xo.js](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/xapi-object-to-xo.js#L146) ?

Reply:
`iscsi_iqn` is only supported by XenServers >= 7.5. So we cant be based on it because we support XenServers >= 5.
Simply call `_updateObjectMapProperty` in `setHostProperties`.

Reply:
why not leave it like this and let `setHostProperties` call only 
`_setObjectProperties` ? 
Please can you review the latest changes?

Reply:
This method only update a property in `other_config`. What if we want to update another property in the `other_config`? should we create another method for that?  This behavior is also done in [setPoolProperties()](https://github.com/vatesfr/xen-orchestra/pull/4208/files#diff-0d7c8183f21c8290ffbeb18bca5c0f88R288).